src/Phinx/Db/Adapter/SqlServerAdapter.php

```diff
@@ -1176,7 +1176,7 @@ public function createDatabase(string $name, array $options = []): void
     {
         $databaseName = $this->quoteColumnName($name);
         if (isset($options['collation'])) {
-            $this->execute(sprintf('CREATE DATABASE %s COLLATE [%s]', $databaseName, $options['collation']));
+            $this->execute(sprintf('CREATE DATABASE %s COLLATE %s', $databaseName, $options['collation']));
```

Comment:
Not sure why the collate argument had brackets around it as it's not an identifier and shouldn't be created as such. I also tried `'` and `"` and it didn't work either.

This has probably been broken for a long time, and I guess no one is creating databases with specific collations.